west.yml

```diff
@@ -82,7 +82,6 @@ manifest:
           - loramac-node
           - lvgl
           - lz4
-          - mbedtls
```

Comment:
when removing this, should we update this to be placed at the location where Zephyr would have placed this ?
Like we do for other projects where we have our own fork, example: mcuboot.

https://github.com/nrfconnect/sdk-nrf/blob/6e0ce2fb2be41372d6c0048b69db8d0464da6f99/west.yml#L112-L113

and change the name to `mbedtls` instead of `mbedtls-nrf`.

Reply:
Agreed with Carles to do this later in a modules re-organization.

Change the location I think is a larger change than just the manifest, the path is most likely hardcoded.

Reply:
accepted.